compiler/src/iree/compiler/Pipelines/Pipelines.cpp

```diff
@@ -146,10 +147,10 @@ void buildIREEVMTransformPassPipeline(
   if (compileTo == IREEVMPipelinePhase::ABI)
     return; // early-exit
 
-  IREE::Flow::TransformOptions flowOptions;
-  flowOptions.constExprHoisting =
+  GlobalOptimization::TransformOptions globalOptOptions;
```

Comment:
I think we want to keep the `IREE::Flow::TransformOptions`. It might not be used for anything now, but might have a use later. For example, all the demotion stuff probably needs to be in `IREE::Flow::TransformOptions`.

Reply:
Move them to GlobalOptimization method, so we still no longer need it.

Reply:
We might not need it now, but maybe leave it as a placeholder for future?

Reply:
It is quite weird to me because the struct does no have any members. I can add it back though.